acp/types/types.go

```diff
@@ -152,6 +154,8 @@ resources:
         expr: owner + admin
       nac-relation-delete:
         expr: owner + admin
+      schema-add:
```

Comment:
suggestion: I think `PatchCollection` should also use this permission, as long term users will be able to add collections via patches (artificially disabled atm).

There is perhaps a case for splitting up the permissions within `Patchollection`, dpending on what exactly the user is patching, but that complication can probably be introduced later if we want.

This (these?) permission(s) will probably also need to be checked during the P2P sync of collection versions (when we add the feature later this sprint).